### PR TITLE
Fit and Finish UX Fixes

### PR DESCRIPTION
**Pull request by jowg-amazon**
### Description
Addresses the fit and finish UX changes. Changes in this PR include:
- Fixes the redirect from Notifications button so that breadcrumbs reflects the correct path
- Adds tooltip to delete icon button
- Moves the `edit` action under `actions` dropdown in the monitors page
- Changes so that view alerts in the datasource alerts card opens in the same tab
- makes the spacing between the header and panel consistent (16px) across all pages
- makes the spacing around the content consistent (16px) across all pages
- Moves pagination to the bottom of the table
- Changes `Updated monitor` to `Edit monitor` in the title
- Changes `Update monitor` to `Save`
- Removes the bottom border in the empty state for the alerts and monitors pages
- Adds a plus icon to the create monitor button

Alerts Page
Screenshot 2024-09-16 at 12 00 16 PM

Monitors Page
Screenshot 2024-09-16 at 12 01 03 PM

Empty Monitors Page
Screenshot 2024-09-16 at 12 03 49 PM

View Monitor Page
Screenshot 2024-09-16 at 12 01 27 PM

Edit Monitor Page
Screenshot 2024-09-16 at 12 01 41 PM

### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/pages/Dashboard/components/DashboardControls/DashboardControls.js [hunk 1]
```diff
@@ -87,9 +88,6 @@ const DashboardControls = ({
       {alertActions.map((action, idx) => (
         <EuiFlexItem grow={false}>{action}</EuiFlexItem>
       ))}
-      <EuiFlexItem grow={false} style={{ justifyContent: 'center' }}>
-        <EuiPagination pageCount={pageCount} activePage={activePage} onPageClick={onPageChange} />
```
Comment (amsiglan) [inline public/pages/Dashboard/components/DashboardControls/DashboardControls.js:91]: We don't need pagination anymore?

Comment (amsiglan) [reply public/pages/Dashboard/components/DashboardControls/DashboardControls.js:91]: I see this was moved to Dashboard.js, is that right?

File: public/pages/MonitorDetails/containers/MonitorDetails.js [hunk 2]
```diff
@@ -492,12 +492,15 @@ export default class MonitorDetails extends Component {
...
     if (useUpdatedUx) {
       monitorActions.unshift(
-        <EuiSmallButton
-          iconType={'trash'}
-          onClick={this.onDeleteClick}
-          color="danger"
-          aria-label="Delete"
-        ></EuiSmallButton>
+        <EuiToolTip content="Delete">
+          <EuiSmallButton
```
Comment (amsiglan) [inline public/pages/MonitorDetails/containers/MonitorDetails.js:496]: did we figure out the spacing for this one?

File: public/utils/helpers.js [hunk 3]
```diff
@@ -160,7 +160,9 @@ export function initManageChannelsUrl(httpClient) {
     const relativePath = `/app/${
       getUseUpdatedUx() ? 'channels' : 'notifications-dashboards'
     }#/channels`;
-    MANAGE_CHANNELS_URL = httpClient.basePath.prepend(relativePath);
+    MANAGE_CHANNELS_URL = httpClient.basePath.prepend(relativePath, {
```
Comment (amsiglan) [inline public/utils/helpers.js:163]: Let's verify this is only used in one place and that the functionality doesn't break when new home page is disabled

File: public/pages/Monitors/components/MonitorControls/MonitorControls.js [hunk 4]
```diff
@@ -47,9 +47,6 @@ const MonitorControls = ({
       <EuiCompressedSelect options={states} value={state} onChange={onStateChange} />
     </EuiFlexItem>
     {monitorActions && <EuiFlexItem grow={false}>{monitorActions}</EuiFlexItem>}
-    <EuiFlexItem grow={false} style={{ justifyContent: 'center' }}>
```
Comment (amsiglan) [inline public/pages/Monitors/components/MonitorControls/MonitorControls.js:50]: should this be shown behind the `useUpdatedUX` condition check?

File: public/pages/MonitorDetails/containers/MonitorDetails.js (hunk 2 above)
Comment (jowg-amazon) [reply public/pages/MonitorDetails/containers/MonitorDetails.js:496]: Yes can see what the button looks like under the view monitor page image

File: public/pages/Monitors/components/MonitorControls/MonitorControls.js (hunk 4 above)
Comment (jowg-amazon) [reply public/pages/Monitors/components/MonitorControls/MonitorControls.js:50]: I don't think we need this in MonitorControls at all. I think there was originally duplication in the pagination. We implemented pagination at the bottom here: https://github.com/opensearch-project/alerting-dashboards-plugin/blob/4776c962fc37877400bd56b8cbb74cedff0d02dc/public/pages/Monitors/containers/Monitors/Monitors.js#L514

File: public/utils/helpers.js (hunk 3 above)
Comment (jowg-amazon) [reply public/utils/helpers.js:163]: This is used in the create and edit monitor workflow as the link for the `manage channels` button but it appends the correct base path for when home page is both enabled and disabled.

**Comment by jowg-amazon** [comment]
`Acknowledge_alerts_modal_spec.js` cypress tests passed locally
```

       Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  acknowledge_alerts_modal_spec.js         01:45        3        3        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        01:45        3        3        -        -        -  

```

**Review by AWSHurneyt — APPROVED** [review]
Changes LGTM.

**Review by riysaxen-amzn — APPROVED** [review]

**Pull request merged**